.github/workflows/main.yml

```diff
@@ -16,7 +16,7 @@ jobs:
       - run: yarn lint
       - run: yarn test
       - run: yarn build || { echo "build command should always succeed" ; exit 61; }
-      - run: yarn build --quiet && git diff --quiet action || { echo "action should be auto generated" ; exit 62; }
+  #      - run: yarn build --quiet && git diff --quiet dist || { echo "dist should be auto generated" ; exit 62; }
```

Comment:
should we just remove that line?

Reply:
Yea ideally we do check somehow that the dist files are  generated along with the PR.

Perhaps auto-generate it pre-merge or after every push?

Reply:
I created a separate issue for this line: https://github.com/game-ci/unity-actions/issues/133